tests/unit/rewriters.js

```diff
@@ -1,17 +1,17 @@
 define('tests/unit/rewriters',
     ['tests/unit/helpers'],
-    function(h) {
+    function(helpers) {
 
     describe('rewriter pagination cache', function() {
         it('rewrites pagination stuff',
-           h.injector()
+           helpers.injector()
            .mock('core/settings', {cache_rewriting_enabled: true,
                                    api_url: 'https://foo.com'})
-           .run(['core/urls', 'rewriters', 'route_api_args', 'routes'],
-                function(urls, rewriters, route_api_args, routes) {
+           .run(['core/urls', 'rewriters', 'routes'],
+                function(urls, rewriters, routes) {
 
                 var cache = {};
-                var first_result = cache[urls.api.params('search', {q: 'foo'})] = {
+                var first_result = cache[urls.api.params('search', {q: 'foo', limit: 3})] = {
```

Comment:
I consolidated route_api_args into init.js which isn't included here. I'm reconsidering putting it back for testability